signature-specification.md

```diff
@@ -230,6 +230,28 @@ The signing certificate's public key algorithm and size MUST be used to determin
 | EC                   | 384             | ECDSA on secp384r1 with SHA-384 |
 | EC                   | 512             | ECDSA on secp521r1 with SHA-512 |
 
+### Certificate Requirements
+
+The signing certificate MUST meet the following minimum requirements:
+
+- The signing certificate MUST satisfy the following constraints:
+  - The keyUsage extension MUST be present and MUST be marked critical. The bit positions for digitalSignature MUST be set ([RFC-5280](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.3)).
+  - The extKeyUsage extension MUST be present and its value MUST be id-kp-codeSigning ([RFC-5280](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.12)).
```

Comment:
Recommend removing this requirement, as not all signed artifacts will be "code".

Reply:
The idea here is to not allow the use of certificates (like timestamping, TLS) cert that is not meant for singing artifacts.
Do you have a recommendation on how we can achieve this while removing `id-kp-codeSigning`?

---

>Recommend removing this requirement, as not all signed artifacts will be "code".

What kind of signed artifacts won't be a code, can you please share some examples?

Reply:
Opened an issue to track this https://github.com/notaryproject/notaryproject/issues/130